lib/msal-browser/src/config/Configuration.ts

```diff
@@ -190,6 +194,8 @@ export type BrowserTelemetryOptions = {
      * - appVersion: Version of the application using MSAL
      */
     application?: ApplicationTelemetry;
+
+    client?: IPerformanceClient;
```

Comment:
can we call this something like "telemetryClient" instead? We use "client" for a fairly wide range of things

Reply:
Oh nevermind - didn't see this was a child of the "telemetry" object. This is fine.